pkg/storage/system_intake.go

```diff
@@ -420,6 +421,25 @@ func (s *Store) FetchSystemIntakes(ctx context.Context) (models.SystemIntakes, e
 	return intakes, nil
 }
 
+// FetchSystemIntakesWithReviewRequested queries the DB for all open system intakes where user is requested
+func (s *Store) FetchSystemIntakesWithReviewRequested(ctx context.Context, userID uuid.UUID) ([]*models.SystemIntake, error) {
+	intakes, err := sqlutils.WithTransactionRet(ctx, s, func(tx *sqlx.Tx) ([]*models.SystemIntake, error) {
+		intakes := []*models.SystemIntake{}
+		err := namedSelect(ctx, s, &intakes, sqlqueries.SystemIntakeGRBReviewer.GetIntakesWhereReviewRequested, args{
+			"user_id": userID,
+		})
+		if err != nil {
+			return nil, err
+		}
+		return intakes, nil
+	})
```

Comment:
believe you can remove the transaction (`tx` isn't used)

Reply:
Gah, good call. We don't need to use a transaction here but Clay and I were debating wrapping everything with a transaction or not. I went ahead and removed it for now in [a63184](https://github.com/CMSgov/easi-app/pull/2663/commits/da63184c448b5436f0c07d75efdfaf9b4364cd7f)